model/otel/registry.yaml

```
@@ -37,3 +41,73 @@ groups:
         brief: The version of the instrumentation scope - (`InstrumentationScope.Version` in OTLP).
         examples: ['1.0.0']
         stability: stable
+  - id: registry.otel.sdk.telemetry
```

```suggestion
  - id: registry.otel.sdk
```

Should this match attributes below?

---

Fixed in 833352b